tests/test_utils.py

```diff
@@ -88,10 +101,42 @@ def test_get_audio_numpy_a2b():
         fs=48000,
     )
     wavs_dir = os.path.join(data_dir, "3D-MARCo Impulse Responses/01_Speaker_+90deg_3m")
-    wavs = os.listdir(wavs_dir)
+    wavs = [
+        wavs_dir + "/+90deg_010_OCT3D_1_FL.wav",
+        wavs_dir + "/+90deg_011_OCT3D_2_FR.wav",
+        wavs_dir + "/+90deg_012_OCT3D_3_FC.wav",
+        wavs_dir + "/+90deg_013_OCT3D_4_RL.wav",
+        wavs_dir + "/+90deg_014_OCT3D_5_RR.wav",
+        wavs_dir + "/+90deg_015_OCT3D_6_FLh_1m.wav",
+        wavs_dir + "/+90deg_016_OCT3D_7_FRh_1m.wav",
+        wavs_dir + "/+90deg_017_OCT3D_8_RLh_1m.wav",
+        wavs_dir + "/+90deg_018_OCT3D_9_RRh_1m.wav",
+    ]
+
     wavs.sort()
-    B = np.array(
-        [librosa.load(os.path.join(wavs_dir, w), sr=48000, mono=False)[0] for w in wavs]
-    )
+
+    B = np.array([librosa.load(w, sr=48000, mono=False)[0] for w in wavs])
+
     B = a2b(2, B, a.capsule_coords["OCT3D"])
     assert np.allclose(A, B)
+
+    A = _get_audio_numpy(
+        a.micarray_capsule_clip_ids["Eigenmike"]["impulse_response+90d"],
+        a.dataset,
+        "A",
+        "B",
+        a.capsule_coords["Eigenmike"],
+        N=2,
+        fs=48000,
+    )
+
+    B = librosa.load(
+        wavs_dir + "/+90deg_065_Eigenmike_Raw_32ch.wav", sr=48000, mono=False
```

Comment:
please use `os.path.join` to concatenate the `wavs_dir` and the Eigenmike filename.